src/components/GoogleAnalytics.tsx

```diff
@@ -20,14 +20,14 @@ type WithIgnoreHashChange = GoogleAnalyticsProps & {
   };
 };
 
-export function GoogleAnalytics({
+export const BodyScriptGoogleAnalytics = ({
```

Comment:
In order to make this a non-breaking change, could we keep this component as it is?

Otherwise people currently using the library will have to update their implementation.

It is working as it is for many people and if someone want to opt-in into the regular `script` version they can do it, or they can safely update (looking at you dependabot :eyes:) without having to make any changes.

Reply:
I don't think it is a breaking change if I deprecated it (my perspective)

But if it's breaking change for you, I understand that also, just wanna clarify I deprecate it for the name (for me it looks clearer because the component has specific name instead of generic `GoogleAnalytics`), I am cool with either both

Reply:
Gotcha, you right, I see it now, thanks for the clarification @GervinFung !

And you made a good point about the naming to make it clearer. 

On the other hand, since nothing changed in the existing component I don't see the need to change its name, when users update to the new version they might get a deprecation warning but nothing really changed except the name, and that might be misleading as no changes are really required.

Let's keep it as it is. It is working without issues for lots of people and would not like the version update to be confusing with deprecation warnings, just a new opt-on component they can pick if they choose to load the script as soon as possible over performance.